hack/generate-bindata.sh

```diff
@@ -28,14 +28,12 @@ if [[ ! -d "${KUBE_ROOT}/pkg" ]]; then
 	exit 1
 fi
 
-# kube::golang::build_kube_toolchain installs the vendored go-bindata in
```

Comment:
is this still the case / should that be cleaned up?

Reply:
we don't have a build_kube_toolchain AFAICT - https://cs.k8s.io/?q=build_kube_toolchain&i=nope&files=&repos=kubernetes/kubernetes